action.yml

```diff
@@ -23,9 +23,13 @@ inputs:
       Filtering mode for the reviewdog command [added,diff_context,file,nofilter].
       Default is added.
     default: 'added'
+  fail_level:
+      Exit code for reviewdog when errors are found with severity greater than or equal to the given level [none,any,info,warning,error].
+      Default is `none`.
```

Comment:
```suggestion
fail_level:
    description: |
      Exit code for reviewdog when errors are found with severity greater than or equal to the given level [none,any,info,warning,error].
```

Reply:
Fixes YML error.

Reply:
@gaurav-nelson The `fail_level` input on line 26 needs to be indented two spaces in your suggested change.